spec/models/featured_topic_spec.rb

```diff
@@ -21,11 +21,12 @@
       expect(FeaturedTopic.all.count).to eq FeaturedTopic.definitions.count
     end
 
-    describe "constructs sensible queries for each topic" do
+    describe "constructs sensible queries for each topic (except those that are really just a link)" do
       FeaturedTopic.keys.each do |key|
         it "translates #{key}" do
-          result = FeaturedTopic.new(key).solr_fq
-          expect(result).to match EXPECTED_SHAPE
+          featured_topic = FeaturedTopic.new(key)
+          next unless featured_topic.path.include? 'focus'
+          expect(featured_topic.solr_fq).to match EXPECTED_SHAPE
         end
```

Comment:
better with separate examples ("it" blocks) for path and non-path featured topics. 

Best rspec practice is individual tests for individual things. I know we sometimes violate that for convenience -- or in system/feature specs for performance since each spec example can take a long time to run. But it's best to keep those the exception, and get back to the "rule" of trying to keep rspec "it" examples brief and focused on a specific thing to test, not testing multiple things in one test. It makes the tests a lot more informative and easier to deal with when they fail, among other things. 


See this recent blog post on general topic: https://www.tomdalling.com/blog/mentoring/write-detailed-rspec-example-descriptions/